src/ServiceControl.Config/UI/AdvanceOptions/AdvanceOptionsView.xaml

```diff
@@ -0,0 +1,174 @@
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             x:Name="root"
+             d:DesignHeight="300"
+             d:DesignWidth="300"
+             mc:Ignorable="d">
+    <ScrollViewer IsTabStop="False" VerticalScrollBarVisibility="Auto">
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="10" />
+                <RowDefinition Height="30" />
+                <RowDefinition Height="*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            
+            <Grid.Resources>
+                <Style TargetType="{x:Type TextBlock}">
+                    <Setter Property="FontSize" Value="14px" />
+                </Style>
+            </Grid.Resources>
+
+            <!--Instance Title -->
+            
+            <TextBlock Grid.Row="1"
+                    FontSize="18px"
+                    FontWeight="Bold"
+                    Text="{Binding Name}" Margin="60,0,60,0" VerticalAlignment="Center"/>
+
+            <!-- Maintenance Section-->
+            <StackPanel Grid.Row="2" Margin="60,10,60,0">
+                <StackPanel  Visibility="{Binding MaintenanceModeSupported, Converter={StaticResource boolToVis}}" Margin="0,0,0,30">
+                <Border Margin="0,0,0,20"
+                        BorderBrush="{StaticResource Gray90Brush}"
+                        BorderThickness="0,0,0,1">
+                    <TextBlock FontSize="13px"
+                               FontWeight="Bold"
+                               Text="MAINTENANCE MODE" />
+                </Border>
+
+
+                <TextBlock FontSize="12px" 
+                           TextWrapping="Wrap" 
+                           Visibility="{Binding InMaintenanceMode, Converter={StaticResource boolToVisInverted}}"
+                           Margin="0,0,0,20"
+                >
+                Enter database maintenance mode to access to RavenDB Management Studio.  While in this mode all message processing is disabled and the REST API is unavailable. This will prevent ServicePulse and ServiceInsight connecting to this instance.
+                </TextBlock>
+                
+                <TextBlock FontSize="12px" 
+                           TextWrapping="Wrap" 
+                           Visibility="{Binding InMaintenanceMode, Converter={StaticResource boolToVis}}"
+                           Margin="0,0,0,20"
+                >
+                This instance is in database maintenance mode. All message processing is disabled and the REST API is unavailable.<LineBreak />Launch <Hyperlink Command="{Binding OpenUrl}" CommandParameter="{Binding StorageUrl}">
```

Comment:
Is it worth mentioning again that ServicePulse and ServiceInsight cannot connect to this instance in this state?